src/base/emu-i386/simx86/codegen-x86.c

```diff
@@ -1690,8 +1691,8 @@ arith0:		{
 			// addl OVERR_DS(%%ebx),%%e[sd]i
 			G3M(0x03,modrm,IG->ovds,Cp);
 			if (mode&(MREP|MREPNE)) {
-			    // addl MEMBASE(%%ebx),%%e[sd]i
-			    G3M(0x03,modrm,Ofs_MEMBASE,Cp);
+			    // addl %%[re]bp,%%[re][sd]i
```

Comment:
I don't understand, its still added
and subtracted at places. Why wouldn't
it be used only in a SIB form on load/stores?